src/views-components/main-content-bar/main-content-bar.tsx

```diff
@@ -18,8 +18,9 @@ interface MainContentBarProps {
 
 const isButtonVisible = ({ router }: RootState) => {
     const pathname = router.location ? router.location.pathname : '';
-    return !matchWorkflowRoute(pathname) && !matchVirtualMachineRoute(pathname) &&
-        !matchRepositoriesRoute(pathname) && !matchSshKeysRoute(pathname) && !matchKeepServicesRoute(pathname);
+    return !Routes.matchWorkflowRoute(pathname) && !Routes.matchVirtualMachineRoute(pathname) &&
```

Comment:
It could be written like, so that would be easier to expand this with new routes:
``` 
const routes = [
        Routes.matchWorkflowRoute,
       ...
    ];

const hasMatch = routes.any(match => match(pathname);
return !hasMatch;
```